front/src/applications/operationalStudies/components/MacroEditor/ngeToOsrd.ts

```diff
@@ -123,9 +138,14 @@ const createTrainSchedulePayload = (
     return [originPathItem];
   });
 
+  const trainrunLabels = trainrun.labelIds.map(
+    (labelId) => labels.find((label) => label.id === labelId)?.label
```

Comment:
No, a train doesn't always have a label.

Reply:
But all label IDs in train runs must always have an entry in the dto labels array?

Reply:
I'd suggest using `!` then, that way we'll get an error (which we can debug) in case this assumption ends up being broken, instead of silently ignoring the issue.

Reply:
Ok, this solution also seems the best thank you for this catch.